app/2.0/start/toolbox/deploy.md

```diff
@@ -9,21 +9,43 @@ In this step, you'll deploy your application to the web.
 
 ## Build for deployment
 
-Run the following Polymer CLI command to prepare your
-application for deployment:
-
-    polymer build
-
-This command  minifies the HTML, JS, and CSS dependencies of your application,
-and generates a service worker that precaches all of the dependencies
-of your application so that it can work offline.
-
-The built files are output to the `build/default` folder. The default build contains an unbundled
-build with granular resources suitable for serving via HTTP/2 with server push. 
-
-For more information on build options, see the [documentation for the `polymer build` command](/2.0/docs/tools/polymer-cli#build).
-This includes documentation on generating bundled (concatenated) resources suitable for
-serving from servers or to clients that do not support HTTP/2 server push.
+Type `polymer build` to build your Polymer application for production. 
+
+You can serve different builds of your app to browsers with different capabilities. The Polymer Starter Kit is configured to create three builds:
+
+* A bundled, minified build with a service worker, compiled to ES5 for compatibility with older browsers.
+* A bundled, minified build with a service worker. ES6 code is served as-is. This build is for browsers that can handle ES6 code.
+* An unbundled, minified build with a service worker. ES6 code is served as-is. This build is for browsers that support HTTP/2 push.
+
+These builds are configured in the `builds` object in `polymer.json`, a configuration file in the root project directory. 
+
+{.caption `polymer.json`}
+```
+...
+"builds": [
+  {
+    "preset": "es5-bundled"
+  },
+  {
+    "preset": "es6-bundled"
+  },
+  {
+    "preset": "es6-unbundled"
+  }
+]
+...
+```
+
+The builds will be output to a subdirectory under the `build/` directory as follows:
```

Comment:
Or folder/subfolders...  I think we mostly use "folder" in the docs (and you do below, so probably stay consistent.)